### ISSUE TITLE
Make an article explaining the current state of Perl6 web development

### ISSUE THREAD

**Issue by WildYorkies** [perl6/user-experience#18]
It's not clear to outsiders what's going on in this area. There are a bunch of modules on the modules subdomain for servers, but which of those are out of date / not being developed? 

Is the HTTP::Server role the thing that all servers will adhere to? 

What about HTTP::Router? Does this module work with all Server modules or does it only work with the HTTP::Server::Threaded module like in its example?

An article that basically covers the stuff that this website covers would be helpful: http://www.arewewebyet.org/

- What are the different Perl6 web dev components that are necessary? 
  - server, router, templater?
    - What are the current / active modules for each of these?
    - What are the ways that you can combine all of these components together?
- Is there an all in one solution for web dev? (Bailador, Hiker, etc)
  - Can I replace my Perl5 app (mojo, dancer, catalyst) with one of these?
- Tutorial links for getting started

**Comment by WildYorkies** [comment]
@zoffixznet Do you think we should change this repo to be more of a "State of the Ecosystem" thing? https://github.com/perl6/perl6-most-wanted

I could whip of a draft document in there to show what I'm thinking of. Just outlining the domains and where P6 currently stands with working in them.

**Comment by zoffixznet** [comment]
No opinion.

**Comment by Altai-man** [comment]
I'm 👎 on such change. Most Wanted is Most Wanted and nothing else. If you want to imitate "$langname report", then it will be a different page. Most Wanted can help you to see a void, not already done things.

**Comment by MasterDuke17** [comment]
No opinion on 'Most Wanted', but very much like the idea of a 'State of the Ecosystem' for different problem domains (e.g., web dev, machine learning, emoji displaying).

**Comment by tony-o** [comment]
I agree with @Altai-man on moving it to most-wanted.  I do think that such an article would be a great thing for p6

**Comment by WildYorkies** [comment]
Here is a basic structure to work with: https://github.com/WildYorkies/perl6-state-of-the-ecosystem/blob/master/README.md

Anyone/everyone is encouraged to add/edit this

**Comment by tony-o** [comment]
@WildYorkies I dig it.  If you want any of those HTTP::* modules that are under my personal github that should reside under the `perl6` namespace for better oversight (as HTTP::Server and HTTP::Server::[Async/Threaded] all are now), let's discuss -
